classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def intitialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @student.push(student)
+    student.classroom = self
+  end
+end
```

Comment:
- [ ] Please it will be nice to update your `student class` to implement the `belong-to` side of the association(a student belongs to a classroom), like so: 
` def classroom=(classroom)`
      ` @classroom = classroom`
      ` classroom.students.push(self) unless classroom.students.include?(self)`
  `end`